src/components/Posts/IndividualPost/IndividualPost.js

```diff
@@ -96,6 +97,7 @@ export const IndividualPost = () => {
           {post.content}
         </Typography>
       </div>
+      <CommentBox postID={post.postID} />
```

Comment:
- Is that only one comment will be allow for each post? If yes, then you may need to implement some logic to not allow create a second comment

Reply:
This means that each `individualPost` will have one`CommentBox` i.e. `textarea` for users to write comments on it